.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Issues connected
+
+Please provide a list of issues connected to this PR (resolves / closes or ref if not closing the selected issue)
+
+## Description
+
+Please include a summary of the changes
+
+## Pull request type
+
+- [ ] Bug fix
+- [ ] New feature
```

Comment:
Maybe we could add another item for dev-ops related PRs (like the one we're looking at currently)? Or would those fall under `New feature`/`Bug fix` or `Other`?

Reply:
I'm really open to all suggetions - these are the things that came to my mind, but will be happy to add more, especially to make `Other` less reliable